web/src/views/DandisetLandingView/DandisetLandingView.vue

```diff
@@ -270,6 +277,10 @@ export default {
     ...mapState('girder', {
       selected: (state) => state.currentDandiset,
     }),
+    ...mapState('publish', {
+      mostRecentDandiset: (state) => state.mostRecentDandiset,
+      versions: (state) => state.versions,
+    }),
```

Comment:
This can be simplified: https://vuex.vuejs.org/guide/state.html#the-mapstate-helper

```suggestion
    ...mapState('publish', [
      'mostRecentDandiset',
      'versions',
    ]),
```